dangerfile.js

```diff
@@ -0,0 +1,181 @@
+/* eslint-disable no-console */
+// inspire by reacts dangerfile
+// danger has to be the first thing required!
+const { danger, markdown } = require('danger');
+const { exec } = require('child_process');
+const { loadComparison } = require('./scripts/sizeSnapshot');
+
+const parsedSizeChangeThreshold = 300;
+const gzipSizeChangeThreshold = 100;
+
+/**
+ * executes a git subcommand
+ * @param {any} args
+ */
+function git(args) {
+  return new Promise((resolve, reject) => {
+    exec(`git ${args}`, (err, stdout) => {
+      if (err) {
+        reject(err);
+      } else {
+        resolve(stdout.trim());
+      }
+    });
+  });
+}
+
+const upstreamRemote = 'danger-upstream';
```

Comment:
Should we capitalize constants?

```suggestion
const UPSTREAM_REMOTE = 'danger-upstream';
```

Reply:
I don't have a strong opinion about that casing conventions (personally I try to avoid camelCase as much as possible but I value consistency more and the consensus is pretty clear in the JS ecosystem).

I'll change it. Conveys meaning better.